tsconfig.json

```diff
@@ -57,7 +57,7 @@
     // "outFile": "./",                                  /* Specify a file that bundles all outputs into one JavaScript file. If 'declaration' is true, also designates a file that bundles all .d.ts output. */
     "outDir": "./out",                                   /* Specify an output folder for all emitted files. */
     // "removeComments": true,                           /* Disable emitting comments. */
-    // "noEmit": true,                                   /* Disable emitting files from a compilation. */
+    "noEmit": true,                                      /* Disable emitting files from a compilation. */
```

Comment:
I don't understand this. As far as I am aware setting this to true makes it so when you compile TS it doesn't output anything.

Is there a reason for that?

Reply:
![image](https://github.com/user-attachments/assets/26eae2b1-e06e-4812-9a78-cbe4ec0e3318)
Yeah, so I had to toggle this in order to import with TS extensions.

perhaps we could have separate tsconfig files for testing & the src...

Reply:
Can't we just import it with .js extensions? That's what I do all the time and it works for me

Reply:
I'm trying to get it to work, but it seems to be struggling to properly infer things...
Let me try again, last time I went for it was last night